spec.md

```diff
@@ -490,7 +492,9 @@ Accept: application/json
 Authorization: Signature keyId="did:key:z6MkpBMbMaRSv5nsgifRAwEKvHHoiKDMhiAHShTFNmkJNdVW#z6MkpBMbMaRSv5nsgifRAwEKvHHoiKDMhiAHShTFNmkJNdVW" ...
 ```
 
-Example success response:
+Note the trailing `/`
```

Comment:
I think you meant this to be in the other section for `GET /space/{space_id}/`

Reply:
Moved, take another look.